client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -49,9 +50,21 @@ class EditorRevisionsList extends PureComponent {
 	}
 
 	render() {
+		// NOTE: This supports revisions that have been hydrated with author
+		// info (`author` is an object) and the ones that haven't (author is a
+		// string, containing just the ID ).
+		const usersId = uniq( map( this.props.revisions, r => get( r, 'author.ID', r.author ) ) );
```

Comment:
> Wouldn't a selector handling this make more sense?

It feels weird to have such a specific selectors tbh, but at the same time, it also simplify how we get the list of IDs because we don't have to support both hydrated and non-hydrated revisions (the entire purpose of that NOTE comment ;) ) so I introduced a new selector in the latest update :ok_hand: